.cirrus.yml

```diff
@@ -9,33 +9,72 @@ env:
 
 task:
   skip: "changesIncludeOnly('appveyor.yml','CMakeLists.txt','.github/**','docs/**','interface/**','include/wx/{msw,osx,qt}/**','src/{msw,osx,qt}/**','build/{cmake,msw,osx}/**')"
-  name: Cirrus CI / FreeBSD 13 wxGTK 3
-  freebsd_instance:
-    image_family: freebsd-13-0
+  matrix:
+    - name: Cirrus CI / FreeBSD 13 wxGTK 3
+      freebsd_instance:
+        image_family: freebsd-13-0
+      env:
+        osname: FreeBSD
+    - name: Cirrus CI / Debian ARM wxGTK 3
+      arm_container:
+        image: debian:11-slim
+      env:
+        osname: Linux
+        arch: arm
 
   before_script: |
     echo LD_LIBRARY_PATH=`pwd`/lib >> $CIRRUS_ENV
 
     wxPROC_COUNT=`./build/tools/proc_count.sh`
+    # ARM systems have 64 CPUs but run out of memory and crash when using that
+    # many jobs, so limit them to something reasonable.
+    if [ $wxPROC_COUNT -gt 16 ]; then
+      wxPROC_COUNT=16
+    fi
+
     echo wxBUILD_ARGS=-j$wxPROC_COUNT >> $CIRRUS_ENV
 
-    WX_EXTRA_PACKAGES='ccache git' ./build/tools/before_install.sh
+    WX_EXTRA_PACKAGES='ccache git'
+    export WX_EXTRA_PACKAGES
+
+    case "${osname}" in
+      Linux)
+        WX_EXTRA_PACKAGES="$WX_EXTRA_PACKAGES \
+              g++ libexpat1-dev libjpeg-dev libpng-dev libtiff5-dev make zlib1g-dev"
```

Comment:
Nitpick, but I think libtiff should just be `libtiff-dev` now (`libtiff5-dev` is deprecated).

Reply:
Thanks, I didn't know about it. I'll fix it here and also in `docs/qt/install.md`.